image/v1/types.go

```diff
@@ -388,6 +416,14 @@ type ImageStreamTagList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // ImageStreamImage represents an Image that is retrieved by image name from an ImageStream.
+// A not found error will be returned if no such image is referenced by a tag within the
+// ImageStream. Images are created when spec tags are set on an image stream that represent
+// an image in an external registry, when pushing to the integrated registry, or when tagging
+// an existing image from one image stream to another. The name of an image stream image is
+// in the form "<STREAM>@<DIGEST>", where the digest is the content addressible identifier
+// for the image (sha256:xxxxx...). You can use ImageStreamImages as the from.kind of an
+// image stream spec tag to reference an image exactly. The only operations supported on
+// the imagestreamimage endpoint are retrieving the image.
```

Comment:
i think we need more clarification (somewhere) on the distinction between an ImageStreamImage and an Image resource.

Reply:
Added a second sentence.